app/views/holds/_holds_not_ready_table.html.erb

```diff
@@ -35,5 +35,32 @@
     </tbody>
 
   </table>
-  <%= submit_tag('Cancel Selected Holds') %>
+  <div class="row">
+    <div class="col-10">
+      <div class="card">
+        <h5 class="card-header">What to do with selected holds</h5>
+        <div class="row">
+          <div class="col-8">
+            <div class="card-body">
+              <h5>Edit Selected Holds</h5>
+              <%= label_tag 'pickup_library', 'Pickup at:' %>
+              <%= select_tag 'pickup_library', options_for_select(render_pickup_libraries,
+                                                                  disabled: 'Not set', selected: 'Not set') %>
+              <%= label_tag 'hold_expiration_date', 'Not needed after:' %>
+              <%= date_field_tag 'hold_expiration_date' %>
+              <p class="card-text"><small class="text-muted">The date you enter is the date that the system will stop trying to obtain the item for you.  It is not a guaranteed delivery date.</small></p>
+              <br>
+              <%= submit_tag('Update Selected Holds', class: 'btn btn-primary', formmethod: 'PATCH') %>
+            </div>
+          </div>
+          <div class="col-4 border-left">
+            <div class="card-body">
+              <h5 class="card-title">Cancel Selected Holds</h5>
```

Comment:
I think it adds a visual hierarchy that's needed, two halves of the panel have equal headings. [Ruth did review it in the issue and liked it](https://github.com/psu-libraries/myaccount/issues/62#issuecomment-577758899) fwiw. I understand where you're coming from from a pure logic standpoint, but I think from a design perspective it is better. Plus it's important  to really drive home that this is cancelling holds (a destructive action).

Reply:
I see your point, I guess I wished at least the text would be different or sth, but this is not my call and this is more of a next milestone thing rather than functionality so I'm fine with it and all good for this PR :)